test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities

```

Comment:
@istoimenova 
I think for cover behavior described in CRQ enough next test cases:
001, 003, 036, 040, 041, 042, 043 and some test cases for check  "store the value of "steeringWheelLocation" received from HMI internally and keep value during one ignition cycle"

Other test cases are perform checks for supported functionality to this CRQ and I propose move them to separate regression test set :
 - validation of HMI responses functionality (TC: 002 - 034, except mentioned above)
 - traceability of HMI result codes to mobile application functionality (TC: 046 - 057)
- etc